ansible/roles/operator-pipeline/templates/openshift/tasks/digest-pinning.yml

```diff
@@ -99,10 +99,10 @@ spec:
         CSVFILE=$(find $BUNDLE_PATH -name "*clusterserviceversion.yaml" -o -name "*clusterserviceversion.yml")
         RELATED_IMAGE_COUNT=$(yq -e '.spec.relatedImages | length' $CSVFILE)
 
-        if [[ $REPLACEMENT_COUNT == $RELATED_IMAGE_COUNT ]]; then
+        if [[ $RELATED_IMAGE_COUNT >= $REFERENCE_COUNT ]]; then
```

Comment:
What are the contents of the file references.json? I guess it's different than replacements.json, but it's not documented anywhere, not even in the repository https://github.com/containerbuildsystem/operator-manifest.
Can the length of this file equal 0 for the proper submission? If yes, then we won't be able to validate, whether the key `.spec.relatedImages` exists.

Reply:
@lcarva, @MartinBasti would it be possible to update the `operator-manifest` docs with some info about this `references.json` file or does that already exist?

Reply:
It's just an array of strings, e.g.:
```
[
  "registry.example.com/org/spam:latest",
  "registry.example.com/org/ham:1.2"
]
```

Reply:
It's just in CLI help, so example wouldn't hurt. I can add it there.

```
operator-manifest replace --help

...

  REPLACEMENTS_FILE  The path to the replacements file. The format of this file is a simple JSON object where each attribute is a string representing the original image reference and the value is a string representing the new value for
                     the image reference. Use - to specify stdin.
```

Reply:
https://github.com/containerbuildsystem/operator-manifest/pull/44